src/debugMain.js

```diff
@@ -611,7 +611,11 @@ class AndroidDebugSession extends DebugSession {
         var srcfolder = path.dirname(srcfpn);
         var pkginfo;
         for (var pkg in this.src_packages.packages) {
-            if ((pkginfo = this.src_packages.packages[pkg]).package_path === srcfolder) break;
+            var pkgpath = (pkginfo = this.src_packages.packages[pkg]).package_path;
+            pkgpath = pkgpath.toLowerCase();
+            srcfolder = srcfolder.toLowerCase();
+            if (pkgpath === srcfolder)
```

Comment:
Converting to lowercase will fix the problem for Windows, but will break on Linux platforms (where file paths must be compared in a case-sensitive way).

I think the path comparison needs to be OS-dependant. I suggest using [String.localeCompare](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) with different flags for Windows and non-Windows platforms. Something like:

```javascript
// make path comparisons case-insensitive on Windows platforms and case-sensitive on other platforms
var sensitivity = process.platform === 'win32' ? 'base' : 'variant';
for (....) {
   var pkgpath = (pkginfo = this.src_packages.packages[pkg]).package_path;
   if (pkgpath.localeCompare(srcfolder, undefined, {sensitivity}) === 0)
       break;
}
```
On Windows, the `sensitivity`  option will be set to `base`, making it a case-insensitive comparison, otherwise the option will be set to `variant` making it case-sensitive. This should fix it for all platforms.